src/tokenizer/index.ts

```diff
@@ -97,17 +93,157 @@ export default class Tokenizer {
         this.setSettings(this.settings)
         this.onLoad() // Call on load
       }
-
       // Add iframe
       this.container = el
       this.container.appendChild(this.iframe)
+      this.initApplePay(el)
     })
   }
 
   // Only here because it used originally here from the begining
   // DO NOT REMOVE, some clients still call create even though it doesnt do anything anymore.
   public create () { }
 
+  private getApiKeyURL(): string {
+    return this.getTokenizerURL() + '/' + this.apikey
+  }
+
+  private getTokenizerURL(): string {
+    const urlPrefix = this.url
+    if (window.location.href.includes('localhost')) {
+      return localDevUrl.replace(/\/$/, '') + pathUrl
+    } else {
+      return urlPrefix.replace(/\/$/, '') + pathUrl
+    }
+  }
+
+  private initApplePay(container: Element) : void {
+    if (!this.settings?.payment?.types?.includes('apple_pay') && !this.settings?.payment?.applePay) {
+      return
+    }
+
+    document.head.insertAdjacentHTML('beforeend', `
+    <style>
+      apple-pay-button {
```

Comment:
Lets update this so they are inline styles like the Iframe build and not append to document head just in case that brings up issues in the future

Reply:
I like the idea, thanks!